Packs/OpenPhish/Integrations/OpenPhish_v2/OpenPhish_v2.py

```diff
@@ -0,0 +1,223 @@
+    if _is_reload_needed(client, data):
+        reload_command(client)
+        data = get_integration_context()
+
+    url_object_list = []
+    if not data:
+        raise DemistoException("Data was not saved correctly")
+
+    url_list_from_user = argToList(kwargs.get('url'))
+    markdown = "### OpenPhish Database - URL Query\n"
+    urls_in_db = data.get('list', [])
+    for url in url_list_from_user:
+        url_fixed = remove_backslash(url)
+        if url_fixed in urls_in_db:
+            dbotscore = Common.DBotScore.BAD
+            desc = 'Match found in OpenPhish database'
+            markdown += "#### Found matches for given URL " + url + "\n"
+        else:
+            dbotscore = Common.DBotScore.NONE
+            desc = ""
+            markdown += "#### No matches for URL " + url + "\n"
+
+        dbot = Common.DBotScore(url, DBotScoreType.URL, 'OpenPhish', dbotscore, desc)
+        url_object_list.append(Common.URL(url, dbot))
+
+    return CommandResults(indicators=url_object_list, readable_output=markdown)
+
+
+def reload_command(client: Client, **kwargs) -> CommandResults:
+    _save_urls_to_instance(client)
+    return CommandResults(readable_output='updated successfully')
+
+
+def status_command(client: Client, **kwargs) -> CommandResults:
+    data = get_integration_context()
+
+    md = "OpenPhish Database Status\n"
+    if data and data.get('list', None):
+        md += "Total **" + str(len(data.get('list'))) + "** URLs loaded.\n"
+        md += "Last load time **" + timestamp_to_datestring(data.get('timestamp'),
+                                                            "%a %b %d %Y %H:%M:%S (UTC)",
+                                                            is_utc=True) + "**\n"
+    else:
+        md += "Database not loaded.\n"
+
+    return CommandResults(readable_output=md)
+
+
+demisto.debug(f'Command being called is {demisto.command()}')
+
+
+def main():
+    """
+        PARSE AND VALIDATE INTEGRATION PARAMS
+    """
+    # get the service API url
+    base_url = API_PATH
+
+    commands = {
+        'url': url_command,
+        'openphish-reload': reload_command,
+        'openphish-status': status_command,
+    }
+
+    hours_to_refresh = demisto.params().get('fetchIntervalHours', '1')
+
+    try:
+        hours_to_refresh = float(hours_to_refresh)
+        use_ssl = not demisto.params().get('insecure', False)
+        use_proxy = demisto.params().get('proxy', False)
+        client = Client(
+            url=base_url,
+            use_ssl=use_ssl,
+            use_proxy=use_proxy,
+            fetch_interval_hours=hours_to_refresh)
+
+        command = demisto.command()
+        if command == 'test-module':
+            # This is the call made when pressing the integration Test button.
+            result = test_module(client)
+            return_results(result)
+
+        elif command in commands:
+            return_results(commands[command](client, **demisto.args()))
+
```

Comment:
no need for newline in between if/elif
```suggestion
        if command == 'test-module':
            result = test_module(client)
            return_results(result)
        elif command in commands:
            return_results(commands[command](client, **demisto.args()))
```